notebooks/examples/scala/QuickstartNotebook.scala

```diff
@@ -17,7 +17,7 @@ print(s"The raw data is stored in $raw_path")
 
 // MAGIC %md
 // MAGIC ## Enable Mosaic in the notebook
-// MAGIC Mosaic requires Databricks Runtime (DBR) version 10.0 or higher. </br>
+// MAGIC Mosaic requires Databricks Runtime (DBR) version 11.2 or higher. </br>
```

Comment:
@edurdevic same as for automatic-sql-registration

Reply:
Updated